guides/v2.1/cloud/project/project-multi-sites.md

```diff
@@ -14,40 +10,39 @@ functional_areas:
   - Stores
 ---
 
-This topic discusses how to set up {{site.data.var.ee}} to have multiple websites or stores. For example, you might have an English store, a French store, and a German store. For more information on websites, stores, and store views, see [Understanding websites, stores, and store views]({{ page.baseurl }}/cloud/configure/configure-best-practices.html#sites).
+You can configure {{site.data.var.ee}} to have multiple websites or stores, such as an English store, a French store, and a German store. See [Understanding websites, stores, and store views]({{ page.baseurl }}/cloud/configure/configure-best-practices.html#sites). The process to set up multiple stores is as follows:
 
-To set up multiple stores, you must:
-
-1.	[Configure your local installation]({{ page.baseurl }}/config-guide/multi-site/ms_over.html) and test it locally.
-2.	Configure {{site.data.var.ece}} routes and variables.
-3.	Push the changes to an Integration environment and test it.
+1.  Configure and test your local installation.
+1.  Configure {{site.data.var.ece}} routes and variables.
+1.  Push the changes to an Integration environment and test.
 
 ## Configure your local installation
 
 To configure your local installation to use multiple stores, see [Multiple websites or stores]({{ page.baseurl }}/config-guide/multi-site/ms_over.html).
 
-## Configure your Integration environment
-
-After successfully creating and testing multiple stores locally, you must:
+After successfully creating and testing the local installation to use multiple stores, you must prepare your Integration environment:
 
-1.	[Configure routes](#cloud-multi-stores-routes), which specify how incoming URLs are handled by {{site.data.var.ee}}.
-2.	[Set up websites, stores, and store views](#cloud-multi-stores-admin) in your {{site.data.var.ee}} server's {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %}.
-3.	[Modify `magento-vars.php`](#cloud-multi-stores-magento-vars) to specify the values of the `MAGE_RUN_TYPE` and `MAGE_RUN_CODE` variables.
-4.	[Deploy](#cloud-multi-stores-deploy) to your Integration branch and test.
+1.  **Configure routes**—specify how incoming URLs are handled by {{site.data.var.ee}}
+1.  **Set up websites, stores, and store views**—configure using your {{site.data.var.ee}} server Admin
```

Comment:
server Admin? i don't think i've seen the Admin app referred to as that before. I've seen "Admin panel" and "Admin". We should have something in our style guide that dictates how we refer to the Admin UI.

Reply:
Oops! Slip-o-the-lazy...